uint8 id
QuadStateSimple past_positions
QuadStateSimple past_velocities
float64 time_diff
---
bool found_body
QuadState state
